.github/workflows/cmake-multi-platform.yml

```diff
@@ -57,9 +57,9 @@ jobs:
       run: |
         echo "build-output-dir=${{ github.workspace }}/build" >> "$GITHUB_OUTPUT"
     
-    - name: Install libboost (Linux)
+    - name: Install libboost, libmodbus and libpaho-mqtt (Linux)
       if: ${{ matrix.os != 'windows-latest' }}
-      run: sudo apt-get install -y libboost-all-dev
+      run: sudo apt-get install -y libboost-all-dev libmodbus-dev libpaho-mqtt-dev
```

Comment:
May I ask for the reason to not build paho under windows?

Reply:
Fixed.